modules/ocf_mirrors/templates/systemd/sync.service.erb

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Sync service for <%= @title %>
+
+[Service]
+User=<%= @exec_user %>
+Type=simple
+Environment=<%= @environments.map{ | env, val | "\""+env+"="+val+"\"" }.join(' ') %>
```

Comment:
You can (and should, by best practive) have multiple `Environment` options in a systemd service file if you have multiple environment variables to set. I'd change this to be something like this instead, seems much simpler and then if you don't have any environment variables to set, there's no empty `Environment` option either:

```
<% @environments.each do |env, val| -%>
Environment="<%= env %>=<%= val %>"
<% end -%>
```
(I still do like Ruby)